README.md

```diff
@@ -49,5 +49,13 @@ Accessibility is an important part of our project. If you have any questions abo
 Before submitting pull requests, please make sure all static type checks and test cases are passing.
 
 ## License
+
 The Inclusive Coding Environment is open source software, distributed under the [BSD-3](https://github.com/codelearncreate/c2lc-coding-environment/blob/master/LICENSE.txt) license.
 
+## Dependencies
+
+We would like to thank to these amazing open source projects and libraries that Inclusive Coding Environment uses:
```

Comment:
How about "We would like to thank these amazing open source projects and libraries that we use in the Inclusive Coding Environment:"?

Reply:
Thanks @colinbdclark for the review, your edit makes more sense than the original copy 👍